docs/how-to-docs/cert-manager.md

```diff
@@ -211,28 +255,399 @@ spec:
-  metadata:
-    annotations:
-      certmanager.k8s.io/acme-challenge-type: dns01
-      certmanager.k8s.io/acme-dns01-provider: route53
-      certmanager.k8s.io/enabled: "true"
-      kubernetes.io/ingress.class: nginx-internal
-    name: example
-  spec:
-    rules:
-    - host: mysite.example.com
-      http:
-        paths:
-        - backend:
-            serviceName: service_name
-            servicePort: 443
-          path: /
-    tls:
-    - hosts:
-      - mysite.example.com
-      - example.com
-      secretName: example-tls
+kind: Ingress
+metadata:
+  name: {{ .DEPLOYMENT_NAME }}-external
+  annotations:
+    certmanager.k8s.io/acme-challenge-type: "http01"
+    certmanager.k8s.io/enabled: "true"
+    ingress.kubernetes.io/backend-protocol: "HTTPS"
+    ingress.kubernetes.io/force-ssl-redirect: "true"
+    kubernetes.io/ingress.class: "nginx-external"
+spec:
+  rules:
+  - host: {{ .APP_HOST_EXTERNAL }}
+    http:
+      paths:
+      - backend:
+          serviceName: {{ .DEPLOYMENT_NAME }}
+          servicePort: 10443
+        path: /
+  tls:
+  - hosts:
+    - {{ .APP_HOST_EXTERNAL }}
+    secretName: {{ .DEPLOYMENT_NAME }}-external-tls
+```
+
+should be changed to `Ingress` resource with the following v0.11 annotations:
+
+```YAML
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: {{ .DEPLOYMENT_NAME }}-external
+  annotations:
+    # @Note: get rid of any certmanager.k8s.io annotations
+    # @Note: change the enabled annotation to cert-manager.io/enabled
+    cert-manager.io/enabled: "true"
+    ingress.kubernetes.io/backend-protocol: "HTTPS"
+    ingress.kubernetes.io/force-ssl-redirect: "true"
+    kubernetes.io/ingress.class: "nginx-external"
+spec:
+  rules:
+  - host: {{ .APP_HOST_EXTERNAL }}
+    http:
+      paths:
+      - backend:
+          serviceName: {{ .DEPLOYMENT_NAME }}
+          servicePort: 10443
+        path: /
+  tls:
+  - hosts:
+    - {{ .APP_HOST_EXTERNAL }}
+    secretName: {{ .DEPLOYMENT_NAME }}-external-tls
+```
+
+###### Internal Ingress changes
+
+Changes required for websites or services exposed internally
+
+The following `Ingress` resource with v0.8 annotations:
+
+```YAML
+apiVersion: networking.k8s.io/v1beta1
+kind: Ingress
+metadata:
+  name: {{ .DEPLOYMENT_NAME }}-internal
+  annotations:
+    certmanager.k8s.io/acme-challenge-type: "dns01"
+    certmanager.k8s.io/enabled: "true"
+    certmanager.k8s.io/acme-dns01-provider: "route53"
+    ingress.kubernetes.io/backend-protocol: "HTTPS"
+    ingress.kubernetes.io/force-ssl-redirect: "true"
+    kubernetes.io/ingress.class: "nginx-internal"
+spec:
+  rules:
+  - host: {{ .APP_HOST_INTERNAL }}
+    http:
+      paths:
+      - backend:
+          serviceName: {{ .DEPLOYMENT_NAME }}
+          servicePort: 10443
+        path: /
+  tls:
+  - hosts:
+    - {{ .APP_HOST_INTERNAL }}
+    secretName: {{ .DEPLOYMENT_NAME }}-internal-tls
+```
+
+should be changed to `Ingress` resource with the following v0.11 annotations:
```

Comment:
v0.12+?

Reply:
Same as above, I think it should have been v0.11+